apps/talon/talon_debug/talon_debug.py

```diff
@@ -0,0 +1,14 @@
+from talon import Module
+
+mod = Module()
+mod.apps.talon_debug = """
+os: mac
+and app.bundle: com.talonvoice.Talon
+win.title: Talon Debug
+"""
+mod.apps.talon_debug = """
+os: windows
+and app.name: Talon
+os: windows
+and app.exe: talon.exe
```

Comment:
We should probably add a title here as well. Otherwise this will just match to the update window and so on.

Reply:
Good catch.

I just realized this is a beta-only thing though. What's the thinking on committing this?

I'm inclined not to commit, or perhaps at least disable the 'talon debug' with a comment to that affect.

Reply:
Maybe we can just display a message if you try to use it? The mode doesn't do any harm if you don't have the beta.

Reply:
I generally don't like adding global commands that simply don't work. I've removed the relevant command and added a note to each file that it's beta only

Reply:
Sounds good too!